packages/api/schema/stryker-core.json

```diff
@@ -374,6 +377,23 @@
         ]
       }
     },
+    "ArrowFunction": {
+      "title": "ArrowFunction",
+      "type": "array",
+      "description": "Mutates bodies of arrow functions to undefined",
+      "uniqueItems": true,
+      "default": [],
+      "minItems": 1,
+      "items": {
+        "anyOf": [
+          {
+            "const": "BodyToUndefined",
+            "title": "BodyToUndefinedMutator",
+            "description": "Replace body to ```undefined```."
+          }
+        ]
+      }
```

Comment:
In binary cases, where the mutator only has a single operator (this one, BlockStatement, and maybe others) we need to discuss if we should make it a list with a single element (like you did here) or a boolean (like it's done in BlockStatement)